### PR TITLE
#325: fixing bug where the sizes table could appear empty

### PR DESCRIPTION
**Pull request by hani-ibrahim**
working on #325

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDK/APIClient_public.swift [hunk 1]
```diff
@@ -140,7 +140,14 @@ extension APIClient {
             clientId: config.clientId,
             fields: nil)
 
-        fetch(from: endpoint, completion: completion)
+        let fetchCompletion: ArticlesCompletion = { result in
+            if case let .success(article) = result where !article.hasAvailableUnits {
+                completion(AtlasResult<Article>.failure(AtlasCatalogError.outOfStock))
```
Comment (bonkey) [inline AtlasSDK/AtlasSDK/APIClient_public.swift]: Wouldn't be enough to just `completion(.failure(AtlasCatalogError.outOfStock))`?

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**